openldap-mixin/config.libsonnet

```diff
@@ -0,0 +1,21 @@
+{
+  groupLabels: ['job', 'cluster'],
+  instanceLabels: ['instance'],
+  filteringSelector: 'job=~".*openldap.*"',
+  uid: 'openldap',
+  clusterLegendLabel: ['openldap_cluster', 'instance'],
+  // prefix dashboards titles
+  dashboardNamePrefix: '',
+  dashboardTags: [self.uid],
+  dashboardPeriod: 'now-1h',
+  dashboardTimezone: 'default',
+  dashboardRefresh: '1m',
+
+  alertsWarningConnectionSpike: 100,
+  alertsWarningHighSearchOperationRateSpike: 200,
+  alertsWarningDialFailureSpike: 10,
+  alertsWarningBindFailureRateIncrease: 10,
+
+  enableLokiLogs: true,
```

Comment:
do you need any other log configs here like the extra labels one?

Reply:
added other labels

Reply:
Good to resolve